website/templates/project/modal_remove_contributor.mako

```diff
@@ -0,0 +1,122 @@
+<link rel="stylesheet" href='/static/css/pages/remove-contributor-page.css'>
+<div id="removeContributor" class="modal fade">
+    <div data-bind="css: modalSize">
+        <div class="modal-content">
+            <div class="modal-header">
+                <button type="button" class="close" data-dismiss="modal" aria-label="Close"><span aria-hidden="true">&times;</span></button>
+                <h3 class="modal-title" data-bind="text:pageTitle"></h3>
+            </div>
+            <div class="modal-body" >
+                <div data-bind="if: canRemoveNode() && !pageChanged()">
```

Comment:
This is probably a reticket if we want to discuss more broadly with the team and consider adding it to our [JavaScript style guide](http://cosdev.readthedocs.org/en/latest/style_guides/javascript.html). If @sloria is okay with it, though, I think we should use KO comments for control flow on the relevant *new* code added on this PR. I'm not suggesting we change old code here.